head_first_design_patterns/pde-bakk/ch10_jsonparser/include/JsonNode.hpp

```diff
@@ -31,18 +31,22 @@ class JsonNode {
 		JSONList	*list;
 		JSONObject	*object;
```

Comment:
see placement new for unions in handling complex data types in unions so you don't have to use pointers here

Reply:
Will do, thanks for the tip!